### DISCUSSION TITLE
UI Remark 1.7.0 update

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#952] [Q&A] [answered]
I updated the remark in my docker-compose, but the new authorization button did not appear.

In the launch logs, I definitely have a new remark.

remark42 v1.7.0-6cf2849-20210416-16: 53: 25

**Comment by akellbl4** [comment, marked as answer]
I can check it. Could you please provide link to your site?

**Reply by uPagge** [reply]
https://blog.upagge.ru/

**Reply by akellbl4** [reply]
As I can see you have new UI for auth :)

**Reply by uPagge** [reply]
Magic, I can see it now too. But I restarted docker-compose several times, replaced the version there with 1.7.0 with latest. And still there were old links to the entrance. I even cleared the cache in the browser. Very strange.

**Reply by akellbl4** [reply]
🤔
